release.nix

```diff
@@ -0,0 +1,13 @@
+{ compiler ? "ghc884"
```

Comment:
I'd prefer we move away from `release.nix` in our repos, and standardize on just using `default.nix`.

`default.nix` is the default, idiomatic Nix entrypoint. There isn't any value in symlinking `default.nix` to `release.nix` since we aren't using Hydra.